### DISCUSSION TITLE
I have not received my withdrawal after 8 day

### DISCUSSION THREAD

**Discussion by ogasky** [blockscout/blockscout#8967] [Support]
Hello, please my withdrawal from pgnETH to ETH wallet address is yet to reflect after 8 days as against the 7 days. Please someone should help get it sorted out. Thank you and below is the explorer and transaction hash details 
https://explorer.publicgoods.network/tx/0x61a54dbb41e66072c6b31d600c2f906ebc184c3e33d50e37c2adfaeb63b328b1
0x61a54dbb41e66072c6b31d600c2f906ebc184c3e33d50e37c2adfaeb63b328b1

**Comment by sl1depengwyn** [comment]
This question is not related to Blockscout as a blockchain explorer, contact public goods team directly
